pkg/controller/pod.go

```diff
@@ -241,6 +241,9 @@ func (c *Controller) enqueueUpdatePod(oldObj, newObj interface{}) {
 	if oldPod.ResourceVersion == newPod.ResourceVersion {
 		return
 	}
+	if newPod.Spec.HostNetwork {
```

Comment:
hostnetwork pod may also be matched by ingress or egress rules and added to addressset

Reply:
fixed